cli/foundation-models/system/finetune/multimodal-classification/multiclass-classification/mmeft-airbnb-multiclass-classification.sh

```diff
@@ -130,7 +130,7 @@ az ml job stream --name $mmeft_parent_job_name $workspace_info || {
 
 # 6. Create model in workspace from train job output for fine-tuned Multimodal model
 az ml model create --name $finetuned_mmeft_model_name --version $version --type mlflow_model \
- --path azureml://jobs/$mmeft_parent_job_name/outputs/mlflow_model_folder $workspace_info  || {
+ --path azureml://jobs/$mmeft_parent_job_name/outputs/trained_model $workspace_info  || {
```

Comment:
Is there any particular reason to call it "trained_model"? 
It would be a deviation as compared to the rest of the examples. 

Same comment for other places where this renaming is done.

Reply:
in rest of the examples we have called it trained_model. So we are aligining it with rest of the examples